meta-ostro/classes/ostro-image.bbclass

```diff
@@ -280,6 +280,10 @@ NOHDD = "1"
 # the rootfs. WARNING: any change to this value will trigger a
 # rebuild (and re-sign, if enabled) of the combo EFI application.
 ROOTFS_PARTUUID_VALUE = "12345678-9abc-def0-0fed-cba987654321"
+# The second value is needed for the system installed onto
+# the device's internal storage in order to mount correct rootfs
+# when an installation media is still inserted into the device.
+ROOTFS_PARTUUID_VALUE2 = "12345678-9abc-def0-0fed-cba987654320"
```

Comment:
I would have choose a slightly more expressive name, like INT_STORAGE_ROOTFS_PARTUUID_VALUE :-)
But apart from that it looks good.
It seems to point to the need for placing the combo files by default elsewhere than in the EFI partition and instead deploy there only the active one. 
@rojkov did you check that the image still fits on a 4GB card, after adding 30MB?